datamade/la-metro-councilmatic
# Rotate exposed Sentry DSN for staging

## Overview

The Sentry DSN for staging is currently exposed in the codebase. Remove this DSN from public code, and rotate it in the encrypted staging configs. 

Connects https://github.com/datamade/devops/issues/113.

### Checklist


### Notes

To rotate the DSN, I created a new DSN for the existing Sentry project, but left the old one in place so we wouldn't experience logging downtime. Once we merge and deploy this PR we can go ahead and remove the old DSN.

## Testing Instructions

 * Examine the new staging settings file with `blackbox_cat configs/settings_deployment.staging.py.gpg` and confirm the key has been rotated
* Grab the Sentry configuration from the staging settings file, drop it into your development settings file, and trigger an error to confirm the new DSN works
